eng/pipelines/ci.yml

```diff
@@ -137,11 +145,12 @@ extends:
 
     - template: /eng/pipelines/templates/stages/vmr-build.yml@self
       parameters:
-        desiredSigning: ${{ parameters.desiredSigning }}
-        desiredIBC: ${{ parameters.desiredIBC }}
         ${{ if eq(variables.isScheduleTrigger, 'true') }}:
-          scope: full
-        ${{ elseif eq(variables.isPRTrigger, 'true') }}:
+          desiredSigning: Signed
```

Comment:
Do we want to condition this on the specific "signed build on weekends" schedule name in case we add other scheduled builds in the future?

Reply:
Actually, I'm not sure if we can condition the `schedule` itself based on a variable so a separate pipeline might be harder. I'm also not sure what would happen with the compliance tooling in a separate "official" pipeline.